DESCRIPTION

```diff
@@ -28,6 +28,7 @@ Imports:
     ggplot2,
     mlr3 (>= 0.12.0),
     mlr3misc (>= 0.9.2),
+    mlr3spatial,
```

Comment:
Judging from the current changes, `mlr3spatial` could be moved to Suggests I think. What do you think?